src/multipatch/connectivity/edge_transformation.hpp

```diff
@@ -12,6 +12,69 @@
 #include "interface.hpp"
 
 
+/**
+ * @brief Get equivalent coordinate on the target patch. 
+ * 
+ * @tparam CurrentPatch Type for the current patch. 
+ * @tparam Interface Interface type given in EdgeTransformation class. 
+ * 
+ * @param current_coord Coordinate on the current patch. 
+ * @param idx_range_patch_1 Index range on the Edge1 of the Interface. 
+ * @param idx_range_patch_2 Index range on the Edge2 of the Interface. 
+ * 
+ * @return Equivalent coordinate on the target patch. 
+ */
+template <
+        typename CurrentPatch,
+        class Interface,
+        class TargetDim = std::conditional_t<
+                std::is_same_v<CurrentPatch, typename Interface::Edge1::associated_patch>,
+                typename Interface::Edge2::parallel_grid::continuous_dimension_type,
+                typename Interface::Edge1::parallel_grid::continuous_dimension_type>,
+        class CurrentDim = std::conditional_t<
+                std::is_same_v<CurrentPatch, typename Interface::Edge1::associated_patch>,
+                typename Interface::Edge1::parallel_grid::continuous_dimension_type,
+                typename Interface::Edge2::parallel_grid::continuous_dimension_type>>
+Coord<TargetDim> get_equivalent_target_coordinate(
+        Coord<CurrentDim> const& current_coord,
+        IdxRange<typename Interface::Edge1::parallel_grid> const& idx_range_patch_1,
+        IdxRange<typename Interface::Edge2::parallel_grid> const& idx_range_patch_2)
+{
```

Comment:
For the readability I would be tempted to do:
```suggestion
template <
        typename CurrentPatch,
        class Interface,
        class CurrentDim>
auto get_equivalent_target_coordinate(
        Coord<CurrentDim> const& current_coord,
        IdxRange<typename Interface::Edge1::parallel_grid> const& idx_range_patch_1,
        IdxRange<typename Interface::Edge2::parallel_grid> const& idx_range_patch_2)
{
    using CurrentEdge = std::is_same_v<CurrentPatch, typename Interface::Edge1::associated_patch>,
                typename Interface::Edge2,
                typename Interface::Edge1>;
    using TargetEdge = Interface::OtherEdge<CurrentEdge>;
    static_assert(std::is_same_v<CurrentDim, typename CurrentEdge::parallel_grid::continuous_dimension_type);
```

Reply:
But this is not much shorter, so feel free to stick with whichever you find easiest to understand

Reply:
Now, it is inside the class in private. So I dont think that someone will give the wrong template parameters.
So maybe I prefer the version I wrote.